backend/game/scribble/handler/checkAnswer/checkAnswer.go

```diff
@@ -0,0 +1,85 @@
+package checkAnswer
+
+import (
+	"encoding/json"
+
+	"github.com/bseto/arcade/backend/game"
+	"github.com/bseto/arcade/backend/log"
+	"github.com/bseto/arcade/backend/websocket/identifier"
+	"github.com/bseto/arcade/backend/websocket/registry"
+)
+
+// Trying to make a function that handles user's answers and compare to the
+// real answer.
+const (
+	name string = "guess"
+)
+
+type IsCorrect struct{}
```

Comment:
This should be named "API". That way when people use this package, they can refer to it as `checkAnswer.API`